src/app/components/JoinedContests/style.scss

```diff
@@ -0,0 +1,61 @@
+:local {
+
+  .pastContest {
+    width: 100%;
+    padding-left: 27px;
+  }
+  .pastContestTitle {
+    width: 100%;
+    height: 31px;
+    color: black;
+    font-size: 24px;
+    line-height: 30px;
+    font-weight: 600;
+  }
+  .pastContestContent {
+    padding-left: 13px;
+    width: 100%;
+    table {
+      border-collapse: collapse;
+      tr {
+        height: 36px;
+        border-bottom: 1px solid rgba(0, 0, 0, 0.05);
+        th {
+          text-align: left;
+        }
+      }
+    }
+  }
+  .tableNumber {
+    width: 89px;
+  }
+  .tableContest {
+    width: 475px;
+  }
+  .tableRanking {
+    width: 370px;
+  }
+  .tableRating {
+    width: 190px;
+  }
+  .rowContest {
+    th {
+      font-weight: 400;
+    }
+  }
```

Comment:
For all columns, except the contest column, use the minimum width. Let the contest column consume the remaining width.